docs/resources/fqdn_proxy.md

```diff
@@ -12,12 +12,14 @@ description: |-
 Resource for deploying a gateway with a fully qualified domain name.
 A user should have some fully qualified domain name (fqdn) (e.g. example.com), pointing to the specified node working as a gateway, then connect this gateway to whichever backend services they desire, making these backend services accessible through the computed fqdn.
 
+
+
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `backends` (List of String) The backends of the gateway proxy (in the format (http|https)://ip:port), with tls_passthrough the scheme must be https.
+- `backends` (List of String) The backends of the gateway proxy. must be in the format ip:port if tls_passthrough is set, otherwise the format should be http://ip[:port]
```

Comment:
why you put port in `[]`

Reply:
because you can omit the port part in the backend
https://github.com/threefoldtech/zos/blob/9caf720c9b83a87ea1ba549442b4d616c8ebabdb/pkg/gridtypes/zos/gw.go#L17